recipes/tvb-framework/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = 'tvb-framework' %}
+{% set version = '1.5.5' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  md5: 3f501c3f43705440f8953bf24aa6b0e5
```

Comment:
Please remove this we only use `sha256`.

Reply:
Ok